app/navigation/router.js

```diff
@@ -216,6 +217,26 @@ class Router extends React.Component {
         };
     };
 
+    handleLeftDrawerClose = () => {
+        setTimeout(() => {
```

Comment:
If I'm understanding these correctly, this prevents everything in `runAfterInteractions` blocks from running between the `createInteractionHandle` and `clearInteractionHandle` calls right? Why do we need to run the clear in a `setTimeout`?